.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "python.pythonPath": "C:\\Users\\jdadams\\AppData\\Local\\Continuum\\miniconda3\\envs\\gdal\\python.exe",
```

Comment:
Does it make more sense to just .gitignore the entire .vscode folder, seeing as how it's developer-specific?

Reply:
I keep going back and forth since the other settings in this file seem like a good idea to share with everyone.

Reply:
My understanding is that this is how VS Code handles using different virtualenvs/conda environments for different projects. I'm a huge fan of creating separate environments for different projects (or families of projects) to avoid dependency problems.

Perhaps ignoring .vscode is a per-project decision- I can see how it would be useful for some, but others (like this) may be simple enough that it's not necessary, or the relevant bits can be put in the development portion of the readme.

Reply:
you should be able to omit `C:\\Users\\jdadams\\AppData\\Local\\Continuum\\miniconda3\\`

Reply:
The best I've been able to make work is "~\\AppData\\Local\\Continuum\\miniconda3\\envs\\gdal\\python.exe." That makes it user-agnostic but not OS or environment-agnostic. I think that's the best we'll get using conda, which is really a necessity for using GDAL's python bindings on Windows (which I imagine will be the biggest end-user OS). I'll add some developer notes to the readme about ensuring the right environment is set.

Reply:
ideally this will only affect folks that want to develop on this project. 2 macs > 1 windows ;)

Reply:
My 8-year-old MBP at home is weeping silently.